src/tests/unit/tests/background/tab-context-broadcaster.test.ts

```diff
@@ -11,13 +12,14 @@ describe('TabContextBroadcasterTest', () => {
         const testMessage = { someData: 1 } as any;
         const expectedMessage = { tabId: testTabId, ...testMessage } as StoreUpdateMessage<any>;
 
-        const sendMessageToFramesAndTabMock = Mock.ofInstance((tabId, message) => {});
+        const browserAdapterMock = Mock.ofType<BrowserAdapter>(null, MockBehavior.Strict);
```

Comment:
I think there is a slight difference between using `null` and `undefined` on the mocks... If I remember correctly, using `undefined` makes typemoq to use es6 proxies. Not sure if you're intentionally using `null`. I just thought that I'd mention it.

Reply:
Tried both, no impact on this particular test. Codebase uses both in existing tests; null is slightly more common so sticking with it here.